### PR TITLE
Add footer contact-form

### PR DESCRIPTION
**Pull request by martinkarugaba**
I have added the footer contact form as requested in the instructions.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -586,5 +593,79 @@ <h3 class="text-[20px] font-medium text-black">
       </div>
     </section>
     <!--* END ABOUT MYSELF -->
+
+    <!--* footer -->
+    <footer
+      class="flex flex-col justify-between items-center bg-light-blue w-full overflow-hidden px-5 rounded-tl-[90px] before:content-[''] before:h-[700px] before:w-[65%] before:bg-mobile-footer-shapes before:bg-contain before:-z-1 before:absolute before:left-[35%] before:bg-right before:bg-no-repeat before:mt-5"
```
Comment (joskalenda) [inline index.html]: - [ ] No comment on this section, your code follows the project requirements, but we have a few. things to modify here before we move forward, it is about the background image, please take a look at this image below and fix the change
- [ ] The second thing is your text alignment, please see how it is adjusted in the original design and make that change @martinkarugaba 

fig

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @martinkarugaba , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]...✅ Good commit messages
- [ ]...✅ Green linters 
- [ ]...✅ Correct use of GitHub flow 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -9,6 +9,7 @@
     />
     <title>Martin Karugaba</title>
     <link rel="stylesheet" href="./dist/output.css" />
+    <link rel="stylesheet" href="style.css" />
```
Comment (rskinnerc) [inline index.html:12]: [OPTIONAL] 🤓 Seems like you wanted to use this `style.css` file. However, right now it doesn't exist and is throwing a 404.  I strongly recommend you either remove the link or add the file.

File: tailwind.config.js [hunk 3]
```diff
@@ -24,6 +25,8 @@ module.exports = {
     },
     backgroundImage: {
       'hero-shapes': "url('/img/header-shapes-mobile.png')",
+      'mobile-footer-shapes':
+        "url('/img/footer/contact form shapes-mobile@2x.png')",
```
Comment (rskinnerc) [inline tailwind.config.js:29]: [OPTIONAL] 🤓 Good job adding the required background images. I see you are using TailwindCSS, nice effort. However, I suggest you use the relative path `./path-to-media` when linking your assets. That way you keep it consistent between dev environments. Take a look at this [live version](https://raw.githack.com/martinkarugaba/my-portfolio/add-form/index.html) of your work. Some environments have problems handling non-relative paths.

File: index.html [hunk 4]
```diff
@@ -586,5 +598,79 @@ <h3 class="text-[20px] font-medium text-black">
       </div>
     </section>
     <!--* END ABOUT MYSELF -->
+
+    <!--* footer -->
+    <footer
+      class="flex flex-col justify-between items-center bg-light-blue w-full overflow-hidden px-5 rounded-tl-[90px] before:content-[''] before:h-[690px] before:w-[65%] before:bg-mobile-footer-shapes before:bg-contain before:-z-1 before:absolute before:left-[35%] before:bg-right before:bg-no-repeat before:mt-5"
```
Comment (rskinnerc) [inline index.html:604]: [OPTIONAL] ⭐ Great job. It is showing the background images as per the template!! However, I strongly recommend you add [a custom class](https://tailwindcss.com/docs/adding-custom-styles#adding-component-classes) so you keep the HTML a bit cleaner. That is a downside of using Tailwind, you might end up cluttering your HTML with a lot of classes.

**Review by rskinnerc — APPROVED** [review]
Hi @martinkarugaba 👋👋👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

## Highlights
- Your work resembles the Figma Template
- Good usage of GitHub flow
- Linters are passing.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**